MCprep_addon/materials/generate.py

```diff
@@ -1353,29 +1355,6 @@ def matgen_cycles_principled(mat, passes, use_reflections, use_emission, only_so
 	if use_emission:
 		nodeMixEmit.inputs[0].default_value = 1
 
-	# Special Glass
-	if mat.name.startswith("glass") is True:
```

Comment:
Just calling out that this naming mechanism was a workaround to deal with the fact I don't have perfect record of all materials and those which should be reflective. Haven't looked too closely, but maybe this should be kept with an additive condition of "use_reflections"? I can think more on it later.

Reply:
Ah silly me, I see where you implemented `matgen_special_glass` now.